`outputFilePath` の変数仕様を変更する

```
$ yarn lint
yarn run v1.22.10
$ eslint src/ --ext ts

~~~~/src/modules/generateImages.ts
  15:79  warning  'outputFilePath' is defined but never used  @typescript-eslint/no-unused-vars
  57:79  warning  'outputFilePath' is defined but never used  @typescript-eslint/no-unused-vars

✖ 2 problems (0 errors, 2 warnings)

Done in 4.41s.
```

これをなんとかする, 厳密には
+ optionalArgumentに変更
+ コード上ではストレージに保存しない仕様を維持するために呼び出し側のファイル名引数を消す

--- a/src/modules/generateImages.ts
+++ b/src/modules/generateImages.ts
@@ -9,10 +9,10 @@
  * 1枚目の画像の生成関数
  * @param {string} originalImagePath 元画像のファイルパス
  * @param {Number} symmetrylPoint シンメトリーをする軸のX座標
- * @param {string} outputFilePath 画像を出力するファイルパス
+ * @param {string | undefined} outputFilePath 画像を出力するファイルパス
  * @returns {Promise<string>} 書きだした画像のbase64エンコードしたString
  */
-const make1stImage = async (originalImagePath: string, symmetryPoint: number, outputFilePath: string): Promise<string> => {
+const make1stImage = async (originalImagePath: string, symmetryPoint: number, outputFilePath?: string): Promise<string> => {
 	const {Image} = require('image-js');
 	// 既存画像の読み込みはImage.load()
 	// Promise の返却が先なので `await` で待ってから次に行く(でないと書き込みのタイミングがずれてえらいことになる)
@@ -42,7 +42,8 @@ const make1stImage = async (originalImagePath: string, symmetryPoint: number, ou
 
 	// もう一度貼りつけ
 	SymmetricalImage1 = SymmetricalImage1.insert(flippedImage, {x: symmetryPoint + 10, y: 10});
-	// void SymmetricalImage1.save(outputFilePath); // シンメトリー画像を残したければこれのコメントアウトを外す
+
+	if(typeof outputFilePath !== 'undefined') void SymmetricalImage1.save(outputFilePath);
 	return await SymmetricalImage1.toBase64('image/jpeg');
 };
 
@@ -51,10 +52,10 @@ const make1stImage = async (originalImagePath: string, symmetryPoint: number, ou
  * 2枚目の画像の生成関数
  * @param {string} originalImagePath 元画像のファイルパス
  * @param {Number} symmetrylPoint シンメトリーをする軸のX座標
- * @param {string} outputFilePath 画像を出力するファイルパス
+ * @param {string | undefined} outputFilePath 画像を出力するファイルパス
  * @returns {Promise<string>} 書きだした画像のbase64エンコードしたString
  */
-const make2ndImage = async (originalImagePath: string, symmetryPoint: number, outputFilePath: string): Promise<string> => {
+const make2ndImage = async (originalImagePath: string, symmetryPoint: number, outputFilePath?: string): Promise<string> => {
 	const {Image} = require('image-js');
 	const originalImage = await Image.load(originalImagePath);
 	const originalImageDetails = {
@@ -82,7 +83,8 @@ const make2ndImage = async (originalImagePath: string, symmetryPoint: number, ou
 
 	// 右側なので投稿する画像の `width` から 切り抜かれた画像の分を減算し、更に10px分白枠があるのでその分も減らした所に貼りつける
 	SymmetricalImage2 = SymmetricalImage2.insert(croppedImage, {x: SymmetricalImage2.width - croppedImage.width - 10, y: 10});
-	// void SymmetricalImage2.save(outputFilePath); // シンメトリー画像を残したければこれのコメントアウトを外す
+
+	if(typeof outputFilePath !== 'undefined') void SymmetricalImage2.save(outputFilePath);
 	return await SymmetricalImage2.toBase64('image/jpeg');
 };
 

--- a/src/modules/generateImages.ts
+++ b/src/modules/generateImages.ts
@@ -9,7 +9,7 @@
  * 1枚目の画像の生成関数
  * @param {string} originalImagePath 元画像のファイルパス
  * @param {Number} symmetrylPoint シンメトリーをする軸のX座標
- * @param {string | undefined} outputFilePath 画像を出力するファイルパス
+ * @param {string | undefined} outputFilePath `optional` 画像を出力するファイルパス
  * @returns {Promise<string>} 書きだした画像のbase64エンコードしたString
  */
 const make1stImage = async (originalImagePath: string, symmetryPoint: number, outputFilePath?: string): Promise<string> => {
@@ -52,7 +52,7 @@ const make1stImage = async (originalImagePath: string, symmetryPoint: number, ou
  * 2枚目の画像の生成関数
  * @param {string} originalImagePath 元画像のファイルパス
  * @param {Number} symmetrylPoint シンメトリーをする軸のX座標
- * @param {string | undefined} outputFilePath 画像を出力するファイルパス
+ * @param {string | undefined} outputFilePath `optinal` 画像を出力するファイルパス
  * @returns {Promise<string>} 書きだした画像のbase64エンコードしたString
  */
 const make2ndImage = async (originalImagePath: string, symmetryPoint: number, outputFilePath?: string): Promise<string> => {

--- a/src/modules/makeSymmetryImages.ts
+++ b/src/modules/makeSymmetryImages.ts
@@ -1,7 +1,6 @@
 import vision from '@google-cloud/vision';
 import EnvYaml from './envReader';
 import SymmetryImage from './generateImages';
-import path from 'path';
 import fs from 'fs';
 
 /**
@@ -63,10 +62,8 @@ const generateTweetsImage = async (filePath: string): Promise<tweet.containMedia
 
 			// 顔が検出されない場合があるので確認を通す
 			if(faces.length > 0){
-				// ファイルの名前(拡張子なし)を取る(生成する画像のファイル名の決定に使う)
-				const originalImageBaseFilename = path.basename(filePath, path.extname(filePath));
 
-				const tweetContainMediaPromises = faces.map((face, currentNumber): Promise<tweet.containMedia> => {
+				const tweetContainMediaPromises = faces.map((face): Promise<tweet.containMedia> => {
 					return new Promise((resolve) => {
 						const tweetMediaList: tweet.containMedia = [];
 						// 顔の中心座標の算出

--- a/src/modules/makeSymmetryImages.ts
+++ b/src/modules/makeSymmetryImages.ts
@@ -76,8 +76,8 @@ const generateTweetsImage = async (filePath: string): Promise<tweet.containMedia
 
 						// 2枚の画像の生成を平行して実行、全部終わったら画像を `base64` エンコードしたデータが帰ってくる
 						void Promise.all([
-							SymmetryImage.make1stImage(filePath, FaceCenterCoordinate, `${originalImageBaseFilename}_${currentNumber}_1.jpg`),
-							SymmetryImage.make2ndImage(filePath, FaceCenterCoordinate, `${originalImageBaseFilename}_${currentNumber}_2.jpg`),
+							SymmetryImage.make1stImage(filePath, FaceCenterCoordinate),
+							SymmetryImage.make2ndImage(filePath, FaceCenterCoordinate),
 						])
 							.then((generateImages) => {
 								// ツイートの元画像を追加してからシンメトリー画像を突っ込む

